BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php

```diff
@@ -67,6 +67,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('authorize_status_change.yml');
         $loader->load('authorize_edition.yml');
         $loader->load('restore_entity.yml');
+        $loader->load('dashboard.yml');
```

Comment:
@SOLenG : after a new reflexion it's not a so good idea.
This file only presents a parameter (an array) describing the widgets used on the dashboard (not the widget availbales). So if the integrator wants to use other widgets for his project, he just has to override that array in his project conf. By merging it with our, we'll force our widgets to be present, so merge is not a good thing for that param.